test/cluster_resource_test.go

```diff
@@ -137,7 +138,7 @@ func getClusterConfigMap(namespace, name string) *corev1.ConfigMap {
 			"test.data": `apiVersion: v1
 clusters:
 - cluster:
-    certificate-authority-data: WTJFdFkyVnlkQW89
+    certificate-authority-data: Y2EtY2VydAo=
```

Comment:
Can you add a comment describing where this comes from?

Reply:
Done